decidim-admin/app/controllers/decidim/admin/concerns/participatory_process_admin.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    module Concerns
+      # This concern is meant to be included in all controllers that are scoped
+      # into a participatory process' admin panel. It will override the layout
+      # so it shows the sidebar, preload the participatory process, etc.
+      module ParticipatoryProcessAdmin
```

Comment:
I get what you mean but I don't quite like the naming - it doesn't just need the participatory process, but it also includes stuff related to the admin panel. I find the current naming more self-explanatory. @oriolgual ?